### ISSUE TITLE
Support for `rust-analyzer/viewRecursiveMemoryLayout`

### ISSUE THREAD

**Issue by mrcjkb** [mrcjkb/rustaceanvim#11]
### Feature description

https://github.com/rust-lang/rust-analyzer/blob/master/docs/dev/lsp-extensions.md#view-recursive-memory-layout

**Comment by b1nhack** [comment]
Is this feature supported? I don't see any documentation.

**Comment by mrcjkb** [comment]
> Is this feature supported? I don't see any documentation.

No, I haven't had time to come up with a way to implement it that I'm happy with yet.

[ferris.nvim](https://github.com/vxpm/ferris.nvim) has an implementation though, and can be used in tandem with rustaceanvim.

**Comment by b1nhack** [comment]
I will continue to look forward to your implementation :)

**Comment by mrcjkb** [comment]
> I will continue to look forward to your implementation :)

It will probably be quite a while before I get around to it 😔
